components/planner/src/dynamo/planner/defaults.py

```diff
@@ -74,9 +74,9 @@ class SLAPlannerDefaults(BasePlannerDefaults):
 
 
 class VllmComponentName:
-    prefill_worker = "prefill"
+    prefill_worker = "VllmPrefillWorker"
     prefill_worker_endpoint = "generate"
-    decode_worker = "backend"
+    decode_worker = "VllmDecodeWorker"
```

Comment:
The prior change is needed for Planner, as Planner looks for the name set here: https://github.com/ai-dynamo/dynamo/blob/main/components/backends/vllm/src/dynamo/vllm/args.py#L91-L95. If you want, you can change args.py so that the user passes in `prefill_worker_name` and `decode_worker_name` and it defaults to `prefill` and `backend` so things don't break, but the planner deployment overrides it